packages/tasks/src/tasks/image-text-to-text/about.md

```diff
@@ -32,39 +32,57 @@ Vision language models can recognize images through descriptions. When given det
 
 ## Inference
 
-You can use the Transformers library to interact with vision-language models. You can load the model like below.
+You can use the Transformers library to interact with [vision-language models](https://huggingface.co/models?pipeline_tag=image-text-to-text&transformers). Specifically, `pipeline` makes it easy to infer models.
+
+Initialize the pipeline first.
+
+```python
+from transformers import pipeline
+
+pipe = pipeline("image-text-to-text", model="llava-hf/llava-interleave-qwen-0.5b-hf")
```

Comment:
any suggestions for a good example cc @Vaibhavs10? (I'm fine with this small one)

Reply:
we are trying to keep the barrier of entry low with this pages per compute. we could do small internVL2.5 otherwise.